DirectProgramming/DPC++/GraphAlgorithms/all-pairs-shortest-paths/sample.json

```diff
@@ -2,7 +2,7 @@
     "guid": "4F0F5FBD-C237-4A9F-B259-2C56ABFB40D9",
     "name": "all-pairs-shortest-paths",
     "categories": [ "Toolkit/Intel® oneAPI Base Toolkit/oneAPI DPC++/C++ Compiler/CPU and GPU" ],
-    "description": "all-pairs-shortest-paths using Intel® oneAPI DPC++ Language",
+    "description": "All Pairs Shortest Paths uses finds the shortest paths between pairs of vertices in a graph using a parallel blocked algorithm that enables the application to efficiently offload compute intensive work to the GPU.",
```

Comment:
"All Pairs Shortest Paths uses finds the shortest paths" -> "All Pairs Shortest Paths finds the shortest paths"